### ISSUE TITLE
in diffedit, areas outside of the mask were also edited

### ISSUE THREAD

**Issue by kkkcx** [aayushmnit/diffusion_playground#1]
Thanks for your great work! I'm not sure if you noticed that during the editing process, using 'prompt_2_img_diffedit' function, areas outside of the mask were also edited, meaning that the background also changed. However, it seems that the background in the original paper can remain unchanged. May I ask if you know the reason for this?

**Comment by aayushmnit** [comment]
@kkkcx : I did notice this issue in "prompt_2_img_diffedit" function. The reason why its happening is that I am also conditioning the background image latents(rlatents) using a text prompt -  
![image](https://user-images.githubusercontent.com/8498767/232978220-fa6b04e6-25fa-4f99-b05d-2273aa6fbabb.png)

Ideally, we should not be performing diffusion guidance on the background image. 

Realizing this I started using the `inpaint pipeline` which essentially doesn't modify the background image. 

Hope this helps.
